bindings/rascal/models/krr.py

```diff
@@ -7,13 +7,168 @@
     compute_KNM             Compute GAP kernel of a set of structures
     train_gap_model         Train a GAP model given a kernel matrix and sparse points
 """
+from ast import Raise
```

Comment:
wat? Not sure why you would need this =)

Reply:
I think this is an automatic change from vscode :-)